package.json

```diff
@@ -50,6 +50,7 @@
     "sinon": "^1.17.3",
     "travis-after-all": "^1.4.4",
     "validate-commit-msg": "2.6.1",
+    "which": "^1.2.8",
```

Comment:
As discussed at https://github.com/sarbbottam/eslint-find-rules/pull/72#issuecomment-217050170
>- use [which](https://www.npmjs.com/package/which) to find out if `eslint-find-rules` is instaled globally
if yes
 - use `__dirname` to find if the current instance is global or not
 - if global 
   - display error

Should it be handled differently?